css/index.less

```diff
@@ -356,6 +356,7 @@ ul.tags {
   }
   > .dropdown {
     overflow: visible;
+    max-height: 31vh;
```

Comment:
you're right. I moved it into dropdown-menu, but kept it in index.less rich-text-editor, because it's specific to the rich-text-editor